checkpointctl.go

```diff
@@ -74,6 +88,9 @@ func show(cmd *cobra.Command, args []string) error {
 	if err := archive.UntarPath(input, dir); err != nil {
 		return fmt.Errorf("unpacking of checkpoint archive %s failed: %w", input, err)
 	}
-
+	
+	if (fullPaths && !showMounts) {
```

Comment:
Sorry, but almost. If you move it to the top of this function it will be before the first call to `Stat()` and `UntarPath()`. This way no disk access happens if we just need to exit here.